pkgs/tools/security/sequoia-tool/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitLab, rustPlatform, pkgconfig, capnproto, clang, libclang, nettle, openssl, sqlite }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sequoia-tool";
+  version = "0.8.0";
+
+  src = fetchFromGitLab {
+    owner = "sequoia-pgp";
+    repo = "sequoia";
+    rev = "v${version}";
+    sha256 = "19k3vsaz0iqm0qxqadlfsnrnprn4lrb8078mm2if3gq7klb5csjb";
+  };
+
+  nativeBuildInputs = [ pkgconfig clang libclang ];
+  buildInputs = [ capnproto nettle openssl sqlite ];
```

Comment:
To get this building under darwin, add `++ stdenv.lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security` to the `buildInputs`.

Reply:
Thank you!